mojave/items/guns/weapons/shotgun.dm

```diff
@@ -87,7 +90,7 @@
 
 
 /obj/item/gun/ballistic/shotgun/automatic/ms13/battle/sks/scoped
-	name = "Scoped SKS"
+	name = "\improper scoped SKS"
```

Comment:
```suggestion
	name = "scoped SKS"
```
This is not needed, `\improper` is only there to override the first letter of `name` being capitalized.